hw/ip_templates/ac_range_check/rtl/ac_range_check.sv.tpl

```diff
@@ -6,14 +6,20 @@ module ${module_instance_name}
   import tlul_pkg::*;
   import ${module_instance_name}_reg_pkg::*;
 #(
-  parameter logic [NumAlerts-1:0] AlertAsyncOn = {NumAlerts{1'b1}}
+  parameter logic [NumAlerts-1:0] AlertAsyncOn = {NumAlerts{1'b1}},
+  parameter bit                   RaclErrorRsp         = 1'b1,
+  parameter int unsigned          RaclPolicySelVec[${3 + 5*num_ranges}] = '{${3 + 5*num_ranges}{0}}
 ) (
   input  logic                                      clk_i,
   input  logic                                      rst_ni,
   input  logic                                      rst_shadowed_ni,
   // Alerts
   input  prim_alert_pkg::alert_rx_t [NumAlerts-1:0] alert_rx_i,
   output prim_alert_pkg::alert_tx_t [NumAlerts-1:0] alert_tx_o,
+  // RACL interface
+  input  top_racl_pkg::racl_policy_vec_t            racl_policies_i,
```

Comment:
This discussion would be for a later PR, but I suspect at some point, we'll want to pass this type from a parameter, since it can vary in definition with the racl config's "nr_policies."

Reply:
I remember that can even be passed as a parameter 😀